pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
I understand.. but the idea was to have a generic way to present all potential future special release for other organisations. The Maven profile would start with "sigmah4...". The idea is not to confuse those profile names with other profiles like "sigmah-dev" for instance. But we can rename the profile name to "sigmah4handicap-international". Would it be fine in your opinion ?